cfunits/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py install --record record.txt
+$PYTHON test/run_tests.py
```

Comment:
Do you think you can add this test to the recipe directory?  That way conda runs it automatically without this call.